### PR TITLE
chore(plugin-server): use librdkafka producer everywhere

### PR DESCRIPTION
**Pull request by hazzadous**
We say some 10x improvements in the throughput for session recordings.
Hopefully there will be more improvements here as well, although it's a
little less clear cut.

I don't try to provide any improvements in guarantees around message
production here.

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: plugin-server/src/utils/db/hub.ts [hunk 1]
```diff
@@ -194,31 +192,15 @@ export async function createHub(
...
-            await kafkaProducer.producer.send({
-                topic: KAFKA_JOBS,
-                messages: [
-                    {
-                        key: job.pluginConfigTeam.toString(),
-                        value: JSON.stringify(job),
-                    },
-                ],
-            })
-        } catch (error) {
```
Comment (xvello) [inline plugin-server/src/utils/db/hub.ts:207]: ~What's your reasoning around removing that try/catch? What's the new behaviour like?~ Nevermind, it's been moved to KafkaProducerWrapper instead

**Review by xvello — APPROVED** [review]
Looks good, just one unrelated question.

File: plugin-server/src/main/graphile-worker/schedule.ts [hunk 2]
```diff
@@ -50,13 +50,18 @@ export async function runScheduledTasks(
     }
 
     if (server.USE_KAFKA_FOR_SCHEDULED_TASKS) {
```
[the remark this answers is not in the document]
Comment (hazzadous) [reply plugin-server/src/main/graphile-worker/schedule.ts:52]: Yeah I think it's worth removing now tbh, but I'll do in a followup.

[the remark this answers is not in the document]
Comment (xvello) [reply plugin-server/src/main/graphile-worker/schedule.ts:52]: Probably fine to keep just to be sure, but if there's already an envvar for it, I'd like to try disabling it and see how things go.
I'm kind of tired of watching the lag on these two low-traffic topics, if they don't add a lot of value.

**Pull request merged**